File: ProyectosJAVA/lp2softRRHHController/src/pe/edu/pucp/lp2soft/rrhh/dao/AreaDAO.java

package pe.edu.pucp.lp2soft.rrhh.dao;

import java.util.ArrayList;
import pe.edu.pucp.lp2soft.rrhh.model.Area;


public interface AreaDAO {
    public ArrayList<Area> listar();
    public int insertar(Area area);
    public int modificar(Area area);
    public int eliminar(int idArea);
}

File: ProyectosNET/LP2soft/LP2softRRHHController/DAO/AreaDAO.cs
using LP2softRRHHModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2softRRHHController.DAO
{
    public interface AreaDAO
    {
        //cuando trabajemos con lista usar BrindingList
        BindingList<Area> listar();
        int insertar(Area area);
        int modificar(Area area);
        int eliminar(int idArea);

    }
}
